src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/service/NonAssociationMergeServiceIntTest.kt

```diff
@@ -21,19 +25,76 @@ import uk.gov.justice.hmpps.test.kotlin.auth.WithMockAuthUser
 
 @DisplayName("Non-associations merge service, integration tests")
 @AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
-@Import(HmppsAuthenticationHolder::class, NonAssociationsMergeService::class, ClockConfiguration::class, ApplicationInsightsConfiguration::class)
+@Import(JacksonAutoConfiguration::class, HmppsAuthenticationHolder::class, NonAssociationsMergeService::class, ClockConfiguration::class, ApplicationInsightsConfiguration::class)
```

Comment:
how do we know which test classes this needs to be put on? any others??

Reply:
No don't think so

Reply:
how would we know to prevent this sorta thing in future? why is jackson not auto-configured automatically 🤣

Reply:
It is now - our spring boot hmpps lib is pin to a working version